content/source/docs/enterprise/guides/recommended-practices/part3.2.html.md

```diff
@@ -37,7 +37,7 @@ Start moving infrastructure code into version control. New Terraform code should
 
 ## 3. Create Your First Module
 
-[Terraform modules](https://www.terraform.io/docs/modules/usage.html) are reusable configuration units. They let you manage pieces of infrastructure as a single package you can call and define multiple times in the main configuration for a workspace. Examples of a good Terraform module candidate would be an auto-scaling group on AWS that wraps a launch configuration, auto-scaling group, and EC2 Elastic Load Balancer (ELB). If you are already using Terraform modules, make sure you’re following the best practices and keep an eye on places where your modules could improve.
+[Terraform modules](https://www.terraform.io/docs/modules/index.html) are reusable configuration units. They let you manage pieces of infrastructure as a single package you can call and define multiple times in the main configuration for a workspace. Examples of a good Terraform module candidate would be an auto-scaling group on AWS that wraps a launch configuration, auto-scaling group, and EC2 Elastic Load Balancer (ELB). If you are already using Terraform modules, make sure you’re following the best practices and keep an eye on places where your modules could improve.
```

Comment:
Normally we use root-relative URLs for links between sections, rather than absolute URLs. Is there a reason for the "Terraform modules" link here to be an exception to that, or was this just an oversight?

Reply:
Oh right, I meant to fix that. This section came out of something originally hashed out in a google-doc over months, and I think we just missed a bunch of these in the process. I'll add a commit and catch as many as I can find.

Reply:
Gosh, looks like "as many as I can find" was on the order of "about a hundred" 😝